### DISCUSSION TITLE
Configure Headlamp in-cluster to not verify OIDC certificate

### DISCUSSION THREAD

**Discussion by vojtechmares** [kubernetes-sigs/headlamp#2704] [General]
Hi, we at Notino are experimenting with Headlamp for our clusters.

We are trying to connect it to our IdP using OIDC and our instance of Dex.

But, since our Dex is on our intranet `.tst` domain, we have certificate provisioned by our private Certificate Authority, therefore the certificate is not generally trusted. And I would need to provide a CA pubkey to Headlamp or configure it not to verify the CA.

I did not find any configuration for either of those.

Can you please guide me to correct configuration or shall I open a feature request for this? Thanks!

**Comment by joaquimrocha** [comment]
@vojtechmares Thanks for reaching out. I understand the use-case. I think it makes sense to open a feature request.
We are planning to have backend plugins in the future but for now I think we could add a config flag. Can you open a feature request (and provide as much details as possible)?

**Comment by vojtechmares** [comment]
I've opened a feature request as asked. If you think it needs more detail, I will try to add as much as possible.

Also, I'm closing this discussion in favour of the feature request.
